example.env

```
@@ -68,3 +68,4 @@ APTRUST_REMOTE_SETTINGS_AWS_SECRET_ACCESS_KEY=
 
 # Metrics push gateway url 
 PROMETHEUS_PUSH_GATEWAY=http://pushgateway:9091
+CLUSTER_NAMESPACE=test-ns
```

Let's prefix this for clarity. You don't have to use the `get_subset_by_key_stem` (at least I don't think it's necessary yet.
```suggestion
PROMETHEUS_CLUSTER_NAMESPACE=test-ns
```